client/components/contact/ContactForm.jsx

```diff
@@ -49,49 +48,31 @@ const ContactForm = () => {
     });
   }
 
-  // initialize component
+  // Initialize component.
   useEffect(() => {
-    // componentDidMount code goes here...
+    // ComponentDidMount code goes here...
     clearFields();
     if (!!displayFeedbackSuccess === true) {
-      toast.success('We received your message. Our team will contact you at the email address provided.', {
-        position: 'top-right',
-        autoClose: 5000,
-        hideProgressBar: false,
-        closeOnClick: true,
-        pauseOnHover: true,
-        draggable: true,
-        progress: undefined,
-      });
+      toast.success('We received your message. Our team will contact you at the email address provided.', contactSettings.toast.dark);
     }
 
     if (!!openErrorModal === true) {
-      toast.error('We failed to process your message. Please try again later.', {
-        position: 'top-right',
-        autoClose: 5000,
-        hideProgressBar: false,
-        closeOnClick: true,
-        pauseOnHover: true,
-        draggable: true,
-        progress: undefined,
-      });
+      toast.error('We failed to process your message. Please try again later.', contactSettings.toast.dark);
     }
 
     return () => {
-      // componentWillUnmount code goes here...
+      // ComponentWillUnmount code goes here...
```

Comment:
nit: if the first word in the comment is a function, it's fine to keep it lowercase.

Reply:
Eslint is playing games with me here. I tried a few ways to break up the lamba functions but eslint keeps recreating the one-liner and when I change the variable name, i'm exceeding 100 chars lol. Should we eslint ignore these lines?

Reply:
updated as discussed